example/src/pages/guides/index.md

```diff
@@ -12,14 +12,14 @@ Get Started
 To get started with Analytics 2.0 APIs you must first decide whether your application will need
 an OAuth client or a JWT client.
 
-Authentication
+Authentication {#my-authentication}
 ---
 
 There are multiple type of authentication for authenticating with the analytics APIs. General information
 about authenticating with Adobe's APIs is located here. Specific information related to authenticating
 with the analytics APIs is available in the sections below
 
-## OAuth
+## OAuth {#OAuthentication}
```

Comment:
I don't understand why you're changing anchors here. OAuth is a common term. OAuthentication seems to refer to a very specific package. Also, I don't see any links going to this heading.

Reply:
@keharper this is just testing material. not intended to mean anything other than me adding custom links to various places in the meaningless example test docs. :)